src/settings.rs

```diff
@@ -11,6 +11,8 @@ struct Metering {
     pub call: u64,
     #[cornetto(const, 200)]
     pub generate_event: u64,
+    #[cornetto(mut, 200)]
```

Comment:
Change mut to const if you don't want to modify the value in test mod. Otherwise you'll have to add an argument in the _reset

Reply:
ok thanks, let's try...